analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -713,5 +713,26 @@ public void EventReference_SetsNotNull()
             validator.ValidateTag("AfterAdd", x => x.HasConstraint(ObjectConstraint.NotNull).Should().BeTrue());
             validator.ValidateTag("AfterRemove", x => x.HasConstraint(ObjectConstraint.NotNull).Should().BeTrue());
         }
+
+        [TestMethod]
+        public void Await_ForgetsFieldStates()
+        {
+            const string code = @"
+private object field;
```

Comment:
This should be extended (parameterized test?) to the following list
* private object field01;
* private readonly object field02;
* private static object field03;
* private static readonly object field03;
* private volatile object field04;
* private usafe object field05;

Same list with other access modifiers:

* public
* protected
* internal

Reply:
The scope is the await. We don't need retest your PR again.